### ISSUE TITLE
parse error: Invalid numeric literal at line 1, column 14

### ISSUE THREAD

**Issue by mark-cobra** [Iserv-Group/Zabbix-Meraki#6]
![Screenshot from 2021-07-28 15-56-23](https://user-images.githubusercontent.com/58164816/127336460-48da4d16-26c2-4478-8d52-136323b138eb.png)

When i run meraki.sh I get this error when I insert more then 1 network name in the orgs.txt file. With only 1 name everything is going right.

![Screenshot from 2021-07-28 16-07-04](https://user-images.githubusercontent.com/58164816/127336816-4527788c-7157-4ab2-ac9c-d34ce280ac90.png)
Those are not real names but examples

I also tried various attempts with for example: company network 1 \n another company
But I always get the same error

**Comment by zdegraaf** [comment]
I found in my own testing that the company name needs to be exactly correct and if it isn't, it can give out this error because the script continues with the org_id variable empty. While additional orgs that have their spelling correct will get their data retrieved, the org with the incorrect spelling won't and therefore won't be discovered by Zabbix. 

I've added testing for this specific error to the meraki_status.sh script so that it will echo a message that there is a problem with that org before moving onto the next org. If I have the time, I might try to add alerting in Zabbix to this script to better call out error like this.
